to-do-list-react/src/App.jsx

```diff
@@ -0,0 +1,42 @@
+import './App.css';
+import Header from './components/Header';
+import Navbar from './components/Navbar';
+import Todolist from './components/Todolist';
+import Footer from './components/Footer';
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+
+function App() {
+  const [state , setState] = useState('both');
```

Comment:
이 변수명을 보고 이 친구가 어떤 기능을 하는지 한눈에 알아보기는 힘들 것 같습니다..!!
지금 누가 보여지고 있느냐에 대한 상태값을 저장하는 변수이므로! display 등의 이름은 어떠신가요!
```suggestion
  const [display , setDisplay] = useState('both');
```

Reply:
```js
const DISPLAY = {
	TODAY: 1,
	TOMORROW: 2,
	BOTH:3
}
```

"today", "tomorrow", "both" 등의 값들도 이런 식으로 상수 취급하신 후에
```js
setDisplay(DISPLAY.TOMORROW)
```

```js
`
width:${props => props.display === DISPLAY.TOMORROW ? '0' : '100%'};
`
```
이렇게 해보셔도 좋을 것 같습니다!!
today, tomorrow 이외의 값이 추가되는 경우에 유연하게 바꿀 수 있으며
오타로 인한 에러를 줄여줄 수 있기 때문입니다!!

Reply:
오 상수 사용 배워갑니다

Reply:
오와 좋은방법 이네요!!